docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -36,7 +36,7 @@ The ``<participant>`` element has two attributes defined: ``profile_name`` and `
        DomainParticipant's creation.
      - Optional
 
-.. _domainparticipantconfig:
+.. _domainparticipantconfig: 
```

Comment:
Please, remove trailing whitespace. CI should have complained but it failed downloading the repository. I have relaunched the job.